libr/debug/p/debug_native.c

```diff
@@ -1425,8 +1349,7 @@ static RList *r_debug_native_modules_get (RDebug *dbg) {
 	if (list && !r_list_empty (list)) {
 		return list;
 	}
-#endif
-#if __WINDOWS__
+#elif  __WINDOWS__
 	list = r_w32_dbg_modules (dbg);
```

Comment:
`r_w32_dbg_modules` is not defined so the pointer is getting truncated, same on line 1201